stacking-promotions/server.js

```diff
@@ -42,17 +33,26 @@ export const attachEndpointsStackingPromotions = (app, client) => {
         const vouchersArray = req.body.vouchersArray;
         const products = req.body.items;
         const items = mapInputIntoKnownProducts(products);
+
+        const validateStackableParams = {
```

Comment:
we do not have to split preparing `validateStackableParams` into two steps (defining structure + update values). We can do that in one step.

```ts
const validateStackableParams = {
  order: {
    amount: calculateCartTotalAmount(items)
  },
  redeemables: removeDuplicatedPromoObjects(redeemables),
  customer
}

```

the same for redeem endpoint